_allevents/youth-dataquest.html

```diff
@@ -19,38 +19,76 @@
-		<img src="{{page.image}}" width="100%">
-	</div><br><br><br><br>
-	<div class="col-sm-6">
-		<h4>Nulla et metus tellus. Aliquam.</h4>
-		<p>Nunc cursus, ipsum eget interdum laoreet, nisi eros aliquam tellus, vitae blandit leo tortor vel eros. Etiam tincidunt feugiat faucibus. Sed lorem dolor, condimentum ut suscipit non, tempus vel nunc. Sed ultricies lectus vel lectus sagittis, in ornare ante eleifend. Nunc sit amet maximus sem, ut pulvinar purus. Duis vitae urna lectus. Morbi laoreet ligula lacus, sit amet varius diam placerat quis. Nulla ultricies magna nec porttitor efficitur. Aenean tempor hendrerit posuere.</p>
-
-		<p>Integer lacus ligula, lobortis a mi lacinia, imperdiet vehicula nisl. Donec tincidunt nisi non metus facilisis, ac posuere enim pretium. Nulla eget interdum magna, sit amet aliquet diam. Maecenas in ultricies enim, ac vehicula felis. Aenean at ipsum lorem. Fusce auctor scelerisque sapien in sodales. Nulla elementum augue ac augue ornare, at imperdiet eros pretium. Proin sit amet nisl ut felis accumsan hendrerit. Ut fringilla nec libero et sagittis. In pharetra arcu quis blandit tincidunt. Curabitur pellentesque auctor tellus sit amet dictum. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
-		<br>
-		<!--Enter your focus as a buletted list below-->
-		<h4><i class="fa fa-bullseye" aria-hidden="true"></i></i> Focus Area</h4>
-		<ul>
-			<li> Aenean at ipsum lorem. Fusce auctor scelerisque sapien in sodales. Nulla elementum augue ac augue ornare, at imperdiet eros pretium. Proin sit amet nisl ut felis accumsan hendrerit. Ut fringilla nec libero et sagittis.</li>
-			<li> Etiam tincidunt feugiat faucibus. Sed lorem dolor, condimentum ut suscipit non, tempus vel nunc. Sed ultricies lectus vel lectus sagittis, in ornare ante eleifend. Nunc sit amet maximus sem, ut pulvinar purus. Duis vitae urna lectus.</li>
-			<li> Insert text</li>
-			<li> Insert text</li>
-		</ul>
-		<br>
-		<div class="well">
-			<h4><i class="fa fa-map-marker fa-fw" aria-hidden="true"></i> Location</h4>
+        <br>
+        <div class="well">
+			<h5><i class="fa fa-map-marker fa-fw" aria-hidden="true"></i> Location</h5>
 			<ul class="fa-ul">
-				<li><h5>Cape Town: <a href="/contact.htm">{code}bridge</a><br></h5></li>
+				<li><h6>Cape Town: <a href="/contact.htm">{code}bridge</a><br></h6></li>
 			</ul>
-			<h4><i class="fa fa-calendar-o fa-fw" aria-hidden="true"></i> Date</h4>
+			<h5><i class="fa fa-calendar-o fa-fw" aria-hidden="true"></i> Date</h5>
 			<ul class="fa-ul">
-				<h5>{{page.ev-date | date_to_long_string}}</h5>
+				<h6>{{page.ev-date | date_to_long_string}}</h6>
 			</ul>
+			<ul class="fa-ul">
+			<h5><a href="http://goo.gl/forms/b732l93gPSmmBgD12" target="_blank" type="button" class="btn btn-primary"><i class="fa fa-user-plus"></i> RSVP your attendance</a></p></h5>	
```

Comment:
Yikes, that's what happens when I copy and paste and only change the link. lemme fix